.pylintrc

```diff
@@ -21,6 +21,19 @@ disable=C0103,C0201,C0301,C0302,W0105,W0707,W1202,W1203,R0801,E1136,E0611,C0209,
 # W0105: pointless-string-statement, # kept as no harm
 # R0801: similar lines, # too granular
 # W0212: protected-access
+# C0209: consider-using-fstring, # kept as no harm
+# W1514: unspecified-encoding, # kept as no harm
+# R1734: use-list-literal, # kept as no harm
+# R1735: use-dict-literal, # kept as no harm
+# R1732: consider-using-with
+# W0237: arguments-renamed
+# R1733: unnecessary-dict-index-lookup
+# R1710: inconsistent-return-statements
+# R1714: consider-using-in
+# R1729: use-a-generator
+# R0402: consider-using-from-import
+# W0640: cell-var-from-loop
+# C0206: consider-using-dict-items
```

Comment:
All these were introduced between pylint 2.6.0 to 2.11.1. Now we have introduced Tomte and making that upgrade, we do not want to change our linter settings so these must be skipped.